docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -129,6 +129,50 @@ Ajoutez l'attribut `stagger` pour décaler verticalement la deuxième colonne de
 
 :::
 
+### Link Cards
+
+Utilisez le composant `<LinkCard>` pour créer un lien vers différentes pages.
+
+Une `<LinkCard>` nécessite un `title` et un attribut [`href`](https://developer.mozilla.org/fr/docs/Web/HTML/Element/a#href). Vous pouvez optionellement inclure une courte `description` ou d'autres attributs de lien tels que `target`.
```

Comment:
Here both `title` and `href` are attributes so maybe something like this would be more accurate?

```suggestion
Une `<LinkCard>` nécessite les attributs `title` et [`href`](https://developer.mozilla.org/fr/docs/Web/HTML/Element/a#href). Vous pouvez optionellement inclure une courte `description` ou d'autres attributs de lien tels que `target`.
```